samples/web/content/apprtc/constants.py

```diff
@@ -20,4 +20,10 @@
 RESPONSE_UNKNOWN_ROOM = 'UNKNOWN_ROOM'
 RESPONSE_UNKNOWN_CLIENT = 'UNKNOWN_CLIENT'
 RESPONSE_DUPLICATE_CLIENT = 'DUPLICATE_CLIENT'
+
+RESPONSE_INVALID_ARGUMENT = 'ERR_INVALID_ARG'
```

Comment:
We should be consistent in use of the ERR_ prefix. Either all errors have this, or none. I lean toward having the prefix.